packages/@lwc/babel-plugin-component/src/__tests__/index.spec.js

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+const fs = require('fs');
+const path = require('path');
+const { testFixtureDir } = require('jest-utils-lwc-internals');
+const transform = require('./utils/test-transform');
+
+function normalizeError(err) {
+    if (err.code === 'BABEL_TRANSFORM_ERROR') {
+        return {
+            // Filter out the filename and the stacktrace, just include the error message
+            message: err.message.match(/^.*?\.js: ([^\n]+)/)[1],
+            loc: err.loc,
+        };
+    } else {
+        return {
+            name: err.name,
+            message: err.message,
+        };
+    }
+}
+
+describe('fixtures', () => {
+    beforeEach(() => {
+        // Disable console.error because babel logs warnings if the file size is > 500KB
+        const { error: originalConsoleError } = console;
+        // eslint-disable-next-line no-console
+        console.error = jest.spyOn(console, 'error').mockImplementation((...args) => {
+            if (
+                args.length > 0 &&
+                // To be sure we are not letting any other real problems slip
+                args[0]
+                    .toString()
+                    .includes('[BABEL] Note: The code generator has deoptimised the styling of')
+            ) {
+                return;
+            }
+            originalConsoleError(...args);
+        });
+    });
+    afterEach(() => {
+        jest.clearAllMocks();
+    });
```

Comment:
You can set `compact` to `false` to prevent Babel from logging a warning.  

https://github.com/salesforce/lwc/blob/95834fd381291d7afc1d1b606994476072ea3469/packages/%40lwc/compiler/src/transformers/javascript.ts#L39-L41

Reply:
TIL, thanks!